src/app/src/test/kotlin/org/imperial/mrc/hint/integration/DiseaseTests.kt

```diff
@@ -82,4 +85,25 @@ class DiseaseTests : SecureIntegrationTests() {
         val responseEntity = testRestTemplate.getForEntity<String>("/disease/anc/")
         assertSecureWithSuccess(isAuthorized, responseEntity, "ValidateInputResponse")
     }
+
+    @ParameterizedTest
+    @EnumSource(IsAuthorized::class)
+    fun `can delete survey data`(isAuthorized: IsAuthorized) {
```

Comment:
For these tests I've not done the more complete thing of adding a file and then confirming it gets deleted as in the [baseline tests](https://github.com/mrc-ide/hint/pull/153/files#diff-d0e7b6eef5bc7c7f8d5e2a6e82f4fd13R101). That's because to do so would involve uploading the shape file as well as the survey/program file each time. Since all units are under tests, and these use the same logic as the baseline methods, I figured it wasn't worth making the build even slower than it is. So these tests are just checking that the url is valid and secured. But let me know if you think this is not robust enough.

Reply:
Should be fine - there's pretty comprehensive unit tests of all the bits..